package.json

```
@@ -53,6 +53,7 @@
     "sinon": "^2.3.6"
   },
   "dependencies": {
+    "lodash.isequal": "^4.5.0",
```

Do we really need to add this dependency ? we try to keep vast-client with fewer decencies as possible. And when I checked the code we don't need to have `lodash.isequal` which do deeper check inside an object. What changed here is just instead of having a url now we have **plain** object with `id` and `url`.
IMO we should write `isTrackingObjectEqual` which is really simple liek this:

```suggestion
    function isTrackingObjectEqual(obj1, obj2) {
    {
        return obj1.id === obj2.id && obj1.url === obj2.url;
    }
```

---

yes I also agree, lodash.isequal is rather heavy. It is best to do our own custom function for our needs

---

Removed the dependency and added a custom method for checking equality